clients/include/unit.hpp

```diff
@@ -398,4 +398,31 @@ inline void trsm_err_res_check(T max_error, rocblas_int M, T forward_tolerance,
 #endif
 }
 
+template <>
+inline void trsm_err_res_check(rocblas_float_complex max_error,
+                               rocblas_int           M,
+                               rocblas_float_complex forward_tolerance,
+                               rocblas_float_complex eps)
+{
+    trsm_err_res_check<float>(std::abs(max_error), M, std::abs(forward_tolerance), std::abs(eps));
+}
+
```

Comment:
Is this reusable beyond trsm, could rename and say what it does?

Reply:
`rocblas_abs` should be used instead of `std::abs`.